backend/src/friend/friend.controller.ts

```diff
@@ -43,4 +46,99 @@ export class FriendController {
   async getSentRequests(@GetUser() user: User) {
     return { sentRequests: user.friendRequestsSent };
   }
+
+  /*****************************************************************************/
+  // POST
+  /*****************************************************************************/
+  // Send request
+  @Post('sendFriendRequest')
+  async handleSendFriendRequest(
+    @GetUser() user: User,
+    @Body() payload: { userEmail: string },
+    @Res() res: Response,
+  ) {
+    const result = await this.friendService.sendFriendRequest(
+      user,
+      payload,
+      res,
+    );
+    res
+      .status(result.statusCode)
+      .send({ status: result.status, message: result.message });
+  }
+
```

Comment:
In stead of hard coding all the responses, it would be good practice to have a general function that handles them, including status, statusCode, optional data, etc. It would benefit the auth-module as well. We can make an issue for this to do if we have enough time.